Dockerfile

```diff
@@ -0,0 +1,164 @@
+    && mv /tmp/gatk-protected /opt/${gatk_dir_name}-${gatk_version}
+RUN cd /opt/ && unzip /tmp/${maven_package_name}-bin.zip \
+    && rm -rf /tmp/${maven_package_name}-bin.zip LICENSE NOTICE README.txt \
+    && cd /opt/ \
+    && cd /opt/${gatk_dir_name}-${gatk_version} && /opt/${maven_package_name}/bin/mvn verify -P\!queue \
+    && mv /opt/${gatk_dir_name}-${gatk_version}/protected/gatk-package-distribution/target/gatk-package-distribution-${gatk_version}.jar /opt/GenomeAnalysisTK.jar \
+    && rm -rf /opt/${gatk_dir_name}-${gatk_version} /opt/${maven_package_name}
+
+###############
+#Strelka 2.7.1#
+###############
+RUN apt-get update && apt-get install -y \
+    bzip2 \
+    g++ \
+    make \
+    perl-doc \
+    python \
+    rsync \
+    wget \
+    zlib1g-dev
+
+ENV STRELKA_INSTALL_DIR /opt/strelka/
+
+RUN wget https://github.com/Illumina/strelka/releases/download/v2.7.1/strelka-2.7.1.centos5_x86_64.tar.bz2 \
+    && tar xf strelka-2.7.1.centos5_x86_64.tar.bz2 \
+    && rm -f strelka-2.7.1.centos5_x86_64.tar.bz2 \
+    && mv strelka-2.7.1.centos5_x86_64 $STRELKA_INSTALL_DIR
+
+#strelka requires a couple steps to run, so add a helper script to sequence those
+COPY strelka_helper.pl /usr/bin/
+
+###############
+#Varscon 2.4.2#
+###############
+RUN apt-get update && apt-get install -y \
+    default-jre \
+    wget
+
+ENV VARSCAN_INSTALL_DIR=/opt/varscan
+
+WORKDIR $VARSCAN_INSTALL_DIR
+RUN wget https://github.com/dkoboldt/varscan/releases/download/2.4.2/VarScan.v2.4.2.jar && \
+    ln -s VarScan.v2.4.2.jar VarScan.jar
+
+################
+#Samtools 1.3.1#
+################
+RUN apt-get update && apt-get install -y \
+    bzip2 \
+    g++ \
+    make \
+    ncurses-dev \
+    wget \
+    zlib1g-dev
+
+ENV SAMTOOLS_INSTALL_DIR=/opt/samtools
+
+WORKDIR /tmp
+RUN wget https://github.com/samtools/samtools/releases/download/1.3.1/samtools-1.3.1.tar.bz2 && \
+    tar --bzip2 -xf samtools-1.3.1.tar.bz2
+
+WORKDIR /tmp/samtools-1.3.1
+RUN ./configure --enable-plugins --prefix=$SAMTOOLS_INSTALL_DIR && \
+    make all all-htslib && \
+    make install install-htslib
+
+WORKDIR /
+RUN ln -s $SAMTOOLS_INSTALL_DIR/bin/samtools /usr/bin/samtools && \
+    rm -rf /tmp/samtools-1.3.1
+
+################
+#Pindel 0.2.5b8#
+################
+RUN apt-get update && apt-get install -y \
+    bzip2 \
+    wget \
+    make \
+    ncurses-dev \
+    zlib1g-dev \
+    g++
+
+WORKDIR /opt
+RUN wget https://github.com/samtools/samtools/releases/download/1.2/samtools-1.2.tar.bz2 && \
+    tar xvjf samtools-1.2.tar.bz2
+
+WORKDIR /opt/samtools-1.2
+RUN make && make install
+
+WORKDIR /opt
+RUN wget https://github.com/genome/pindel/archive/v0.2.5b8.tar.gz && \
+    tar -xzf v0.2.5b8.tar.gz
+
+WORKDIR /opt/pindel-0.2.5b8
+RUN ./INSTALL /opt/samtools-1.2/htslib-1.2.1
+
+WORKDIR /
+RUN ln -s /opt/pindel-0.2.5b8/pindel /usr/bin/pindel
```

Comment:
Do we need to add pindel2vcf : `RUN ln -s /opt/pindel-0.2.5b8/pindel2vcf /usr/bin/pindel2vcf`